gauss_plot_2d.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            return multivariate_normal.pdf(jnp.array([x, y]), mean=mean, cov=covariance[G])
</pre><p>There is two tab before <code>return</code> make it one </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>